src/components/staking/LiValidator.vue

```diff
@@ -77,6 +89,23 @@ export default {
       default: () => "returns"
     }
   },
+  computed: {
+    status() {
+      if (
+        this.validator.jailed ||
+        this.validator.tombstoned ||
+        this.validator.status === 0
+      )
+        return `Inactive`
+      return `Active`
+    },
+    status_detailed() {
+      if (this.validator.jailed) return `Temporally banned from the network`
+      if (this.validator.tombstoned) return `Banned from the network`
+      if (this.validator.status === 0) return `Banned from the network`
```

Comment:
i don't think this is true - we'll have to improve some of these messages.

Reply:
Why do you think it's not true?

Reply:
i just mean they are not necessarily "banned"

Reply:
We can customize the messages in the new backend.